pylabrobot/heating_shaking/hamilton.py

```diff
@@ -0,0 +1,81 @@
+from pylabrobot.heating_shaking.backend import HeaterShakerBackend
+from pylabrobot.machines.backends.usb import USBBackend
+from enum import Enum
+
+class PlateLockPosition(Enum):
+  LOCKED = 1
+  UNLOCKED = 0
+
+class Command(Enum):
+  INITIALIZE_PLATE_LOCK = "LI"
+  MOVE_PLATE_LOCK = "LP"
+  SHAKE = "SB"
+  STOP_SHAKING = "SC"
+  WAIT_FOR_STOP = "SW"
+
+class RotationDirection(Enum):
+  CLOCKWISE = 0
+  COUNTER_CLOCKWISE = 1
+
+class HamiltonHeatShaker(HeaterShakerBackend, USBBackend):
+  """
+  Backend for Hamilton Heater Shaker devices connected through
+  an HSB
```

Comment:
Not sure what the HSB acronym stands for 😅

Reply:
https://forums.pylabrobot.org/t/add-carriers-to-hamilton-worktable-using-pylabrobot/2985/6 Heater Shaker Box?